### ISSUE TITLE
This is awesome

### ISSUE THREAD

**Issue by Cncninja** [MitchBradley/GeckoBlaster#1]
I'm going to send the files off to have the PCB made, before I do I just wanted to make sure you or someone else does not have some for sale somewhere.  Thank you

**Comment by MitchBradley** [comment]
I will sell you one.  Look on Discord.

**Comment by Donniem70** [comment]
I have looked on Discord but can not find it. Do you have a link? I would like to purchase one also.

**Comment by MitchBradley** [comment]
I have two boards left.  They do not have the rotary encoder, pushbutton, or serial wombat chip.  It is just the PCB, DB25 connector, socket for a ESP32 Mini module, USB-B connector, and two electrolytic capacitors for preventing reset when the WiFi turns on (some ESP32 Mini modules have insufficient capacitance).  I would also throw in an SD shield because I have more of them than I can ever use.  You would have to supply the ESP32 Mini and (optionally) the OLED shield.

The price would be a $35 donation to the FluidNC project.  That includes shipping to US addresses.  If it has to be shipped internationally, I would have to figure out how much that costs.

**Comment by Donniem70** [comment]
Awesome!
I will take one. Send me the link to donate and I will get that done.
Will you see when I donate or do I need to forward you something?
My address is:

308 Rose Bud Ln, Holly Ridge, NC 28445

My phone number is 910-389-7942.
That is my work cell. Easiest way to contact me if you need to.

Thank you.

**Comment by MitchBradley** [comment]
https://github.com/bdring/FluidNC#donations

I will get a notification.  It will probably be Friday before I can ship.

**Comment by Donniem70** [comment]
Okay, that sounds good. As soon as I get home I will make the donation.

**Comment by MitchBradley** [comment]
Got it, thanks.

**Comment by Donniem70** [comment]
Yes sir. No problem.
I may be asking a few questions here and there. I am new to this stuff, but I follow directions pretty well, lol.
I am currently running Sheetcam and mach3. I am not to found of the mach3 so I hope this is a better option.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by jeffrae** [comment]
Hi do you have any boards left
Is there a schematic or build docs
And IT will be International frieght to Brisbane Australia
Thanks Jeff

**Comment by MitchBradley** [comment]
Yes I have a couple of boards left.  DM me on the FluidNC discord http://wiki.fluidnc.com/en/support/discord and we can work something out.

**Comment by facundosala** [comment]
Hello, thank you for your design. I'm having some troubles finding how to use a rotary encoder with FluidNC, would you have a yaml example file as an example?

Thank you!

**Comment by MitchBradley** [comment]
I never followed up on that part of the design.  The rotary encoder is still in my parts bin, never having been soldered into a GeckoBlaster board.

We are working on a new approach for smart pendants, that uses an auxiliary UART to send commands to FluidNC.  There is a version of that which uses an M5 Dial device with a rotary encoder, LCD touch screen, and extra buttons.  See http://wiki.fluidnc.com/en/hardware/official/M5Dial_Pendant for more information, and a link to the code .
